src/components/Navigation.js

```diff
@@ -1,19 +1,26 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
+        <Link to="/">
```

Comment:
Rather than doing this 
```
   <Link to="/contact">
      <div>
         <a href="">Contact</a>
      </div>
   </Link>
```
Do this => 
```
  <Link to="/contact">Contact</Link>
```